### PR TITLE
Clarifies `updateClientValue` flow - #585

### PR DESCRIPTION
**Pull request by knod**
Trying out this new 'WIP' thing. This PR changes `checked` to `value` for `ControlledRadioYesNo`. Maybe it really should be its own PR, but I'm going to keep moving forward on more tweaks. Let me know what you think!

### CODE REVIEW

**Comment by knod** [comment]
I don't want to let this sit around forever. I'll call it on this one and save further improvements for later. Thoughts on what's here, folks?

**Comment by knod** [comment]
Does your approval mean I can still merge it, or do you want to resolve the conversation about `checked` first?

File: src/utils/convertForUpdate.js [hunk 1]
```diff
@@ -0,0 +1,22 @@
+/**
+ *
+ */
+const convertForUpdate = function ({ name, route, value, checked, ...otherProps }) {
+
+  var val = value;
+  if (typeof checked === `boolean`) {
+    val = checked;
```
[the remark this answers is not in the document]
Comment (ethanbb) [reply src/utils/convertForUpdate.js]: Oh actually, I just did a global search for `checked` to see whether it was ever still used when constructing the object to pass to `updateClientValue`, but I may have missed cases where the properties as a whole of a checkbox, including `checked `, were passed. Not sure though.

**Comment by ethanbb** [comment]
If you don't think a change is necessary based on the comment, you can go ahead and merge it.

File: src/utils/convertForUpdate.js (hunk 1 above)
[the remark this answers is not in the document]
Comment (knod) [reply src/utils/convertForUpdate.js]: Yeah, that's what happens.

**Pull request merged**